src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -154,9 +163,8 @@ const renderSelectComponent = (Component, value, onChange) => {
   );
 };
 
-const renderComponent = (Component, value, onChange) => {
+const renderComponent = (Component, value, onChange, onError) => {
```

Comment:
I think we don't need to pass `onError` to component without additional properties because all validations should be defined in component props object. Or I am missing some use case?